lib/features/modeling/cmd/SpaceToolHandler.js

```diff
@@ -24,12 +32,20 @@ SpaceToolHandler.prototype.preExecute = function(context) {
       direction = context.direction,
       movingShapes = context.movingShapes,
       resizingShapes = context.resizingShapes,
-      start = context.start;
+      start = context.start,
+
+      // TODO(@barmac): replace with new Map() once we drop PhantomJS
+      oldBounds = new BoundsMap();
 
   // (1) move shapes
   this.moveShapes(movingShapes, delta);
 
-  // (2) resize shapes
+  // (2a) save old bounds of resized shapes
+  forEach(resizingShapes, function(shape) {
+    oldBounds.set(shape, getBounds(shape));
```

Comment:
Proposal: Let's not use a map here but rather a plain old object that keys by element ID (`ID` => `oldBounds`).

Reply:
Makes sense. I will do that.

Reply:
OK done.